utils/update-next.js

```diff
@@ -1,68 +1,181 @@
+/* eslint-disable no-console */
+
+const { dirname, join } = require('path');
 const { execFileSync } = require('child_process');
+const {
+  existsSync,
+  readdirSync,
+  readFileSync,
+  statSync,
+  writeFileSync,
+} = require('fs');
 
 function exec(cmd, args, opts) {
-  // eslint-disable-next-line no-console
-  console.log({ input: `${cmd} ${args.join(' ')}` });
+  console.log({ input: `${cmd} ${args.join(' ')}`, cwd: opts?.cwd });
   const output = execFileSync(cmd, args, opts).toString().trim();
   console.log({ output });
   console.log();
   return output;
 }
 
-module.exports = async ({ github, context }) => {
-  const oldVersion = require('../examples/nextjs/package.json').dependencies
-    .next;
+module.exports = async ({ github, context } = {}) => {
```

Comment:
Why was this change necessary? These files should only be invoked by the action and so those properties are definitely assigned. You can add a JSDoc comment like I did in the Gatsby updater for better types: https://github.com/vercel/vercel/blob/91b7f6dcd90ecec469b537f5fc358d579262fe52/utils/update-gatsby-fixtures.js#LL7C1-L11C4

Reply:
I wouldn't say "necessary", but I think it's harmless and even helpful so you can test it in the Node.js repl without having to comment out all the github/context stuff run it.